BaseTools/Plugin/OverrideValidation/OverrideValidation.py

```diff
@@ -631,6 +703,15 @@ def path_parse():
         if not os.path.normcase(Paths.RegenPath).startswith(os.path.normcase(Paths.WorkSpace.rstrip(os.sep)) + os.sep):
             raise RuntimeError("Module is not within specified Workspace.")
 
+    if Paths.DeprecatedPath is not None:
+        if not os.path.isfile(Paths.DeprecatedPath):
+            raise RuntimeError("Deprecation path module is invalid")

```

Comment:
In this new flow, is the path displayed anywhere? It will be useful for debugging purposes either the invalid path, or the file with the comment containing the invalid path, is displayed with the error.